src/bin/lfortran.cpp

```diff
@@ -359,6 +359,12 @@ int prompt(bool verbose, CompilerOptions &cu)
                 std::cout << std::setprecision(17) << "(" << r.c64.re << ", " << r.c64.im << ")" << std::endl;
                 break;
             }
+            case (LCompilers::FortranEvaluator::EvalResult::boolean) : {
+                if (verbose) std::cout << "Return type: logical" << std::endl;
+                if (verbose) section("Result:");
+                std::cout << (r.b ? "True" : "False") << std::endl;
```

Comment:
In Fortran you want to print this as:
```suggestion
                std::cout << (r.b ? ".true." : ".false.") << std::endl;
```

Reply:
I am printing it based on what `print` function does

```bash
>>> print *, .true.
True
```

Do you want me to keep this existing implementation or change it to your suggestion?

Reply:
I see. Leave `True` then for now. We can change both later. In fact I think GFortran just prints T/F. I prefer True or `.true.`, but for compatibility we'll at least have to add a compiler option to print just T/F. But that's a separate issue.